db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/SYSSCHEMASRowFactory.java

```diff
@@ -242,32 +242,28 @@ public TupleDescriptor buildDescriptor(
 	}
 
 	private static final String REGULAR_USER_SCHEMA =
-	"SELECT S.* " +
-	"FROM SYS.SYSSCHEMAS as S, SYS.SYSSCHEMAPERMS as P " +
-	"WHERE  S.schemaid = P.schemaid and P.accessPriv = 'y' " +
-			"and P.grantee in (select name from sysvw.sysallroles) \n" +
-	"UNION ALL \n " +
-	"SELECT S.* " +
-	"FROM SYS.SYSSCHEMAS as S " +
-	"WHERE S.authorizationId " +
-	"     in (values current_user, 'PUBLIC' " +
-	"         union all " +
-	"         select name from new com.splicemachine.derby.vti.SpliceGroupUserVTI() as b (NAME VARCHAR(128)))";
+			"SELECT S.* " +
+					"FROM SYS.SYSSCHEMAS as S, SYS.SYSSCHEMAPERMS as P " +
+					"WHERE  S.schemaid = P.schemaid and P.accessPriv = 'y' " +
+					"and P.grantee in (select name from sysvw.sysallroles) \n" +
+					"UNION ALL " +
+					"SELECT S.* " +
+					"FROM SYS.SYSSCHEMAS as S " +
+					"WHERE S.authorizationId " +
+					"     in (select name from new com.splicemachine.derby.vti.SpliceGroupUserVTI(1) as b (NAME VARCHAR(128)))";
 
 	private static final String SUPER_USER_SCHEMA =
-	"SELECT S.* " +
-	"FROM SYS.SYSSCHEMAS as S ";
+			"SELECT S.* " +
+					"FROM SYS.SYSSCHEMAS as S ";
```

Comment:
nit: make it aligned or a single line. I think it got worse compared to the previous version.